tests/samplesheet/test_long_reads.csv

```diff
@@ -0,0 +1,5 @@
+study_accession,reads_accession,fastq_1,fastq_2,library_layout,library_strategy,platform,assembler,assembly_memory,assembler_config
+ERP118188,ERR3792762,/hps/nobackup/rdf/metagenomics/service-team/users/germanab/miassembler_tests/ERR3792762.fastq.gz,,single,metagenomic,ont,,,
```

Comment:
those paths should be links to sra maybe? `/users/germanab` should be changed

Reply:
Correct thank you, this samplesheet will stay for my tests until I implement one of your solutions!